support/ci/specs.sh

```diff
@@ -56,7 +56,7 @@ tar xvzf ./hab.tar.gz --strip 1 -C ${BOOTSTRAP_DIR}
 # generate a key to use to build `hab` and `hab-sup` in a studio
 # we won't need this after we extract the binaries later in the script.
 # They don't need to be stored past a single Travis run.
-${TRAVIS_HAB} origin key generate hab_travis
+#${TRAVIS_HAB} origin key generate hab_travis
```

Comment:
Nope, meant to uncomment this.  Bash is such an alien scripting environment.  Thanks for the extra 👀